checker/tests/lock/TestTreeKinds.java

```diff
@@ -74,248 +135,343 @@ Object getFoo() {
   @GuardedBy("lock")
   myEnumType myEnum;
 
+/* TODO Reenable this after the Lock Checker is changed to handle primitives.
 public void testOperationsWithPrimitives() {
     int i = 0;
     boolean b;
 
-    //:: error: (contracts.precondition.not.satisfied.field)
+    // TODO reenable this error: (contracts.precondition.not.satisfied.field)
```

Comment:
No, these can only be reenabled once @GuardedByName is implemented. I changed the comment at the top of the method to indicate this.